mattermost-api.cabal

```diff
@@ -42,6 +42,7 @@ library
                      , aeson >= 1.0.0.0
                      , connection
                      , memory <0.14.3
+                     , resource-pool
```

Comment:
Can you please add a lower bound for this dependency? (The latest released major version is okay.)